docs/BR.md

```diff
@@ -305,6 +307,8 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 ### 1.6.1 Definitions
 
+**Address and Routing Parameter Area Name**: A Domain Name whose Top-Level Domain ends with the labels "in-addr.arpa" or "ip6.arpa". Examples: `1.2.0.192.in-addr.arpa` (IP version 4) and `1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.8.b.d.0.1.0.0.2.ip6.arpa` (IP version 6).
```

Comment:
While the definition correctly captures the scope for reverse DNS names, the name "Address and Routing Parameter Area Name" sounds like it's intended to apply to all of `.arpa`. There's a risk of confusion.

Suggest renaming this definition to "IP reverse address domains".

(The change would affect other places in this PR as well.)